site2/website/release-notes.md

```diff
@@ -1,6 +1,30 @@
 
 ## Apache incubator
 
+### 2.0.0-rc1-incubating &mdash; 2018-05-29 <a id="2.0.0-rc1-incubating"></a>
```

Comment:
this should be 2.1.0

Reply:
ah, your are right. I forgot to change it after copying from 2.0.0 :)

Reply:
fixed